### DISCUSSION TITLE
Seting resources for the `dagit` and `daemon` service

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18592] [Q&A]
Hello,
how much resources do you recommend to set for the `dagit` and `daemon` service? a fargate instance of 1CPU + 2 GB Mem for each of those service don't seem to be enough, some groups take forever to appear on the UI whereas locally it's instantly.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C014UDS8LAV/p1702032446532679?thread_ts=1702032446.532679&cid=C014UDS8LAV).*

**Comment by yamhirotoCS** [comment]
any recommendation?

**Comment by ei-grad** [comment]
It depends on how many assets do you have, how many runs could the sensor schedule at one tick, etc. Set it to a higher value which would be enough for sure (e.g. 8Gb), monitor the memory usage, and decrease accordingly on the real need. I use 4G, less was not enough for a dozens of simultaneously-sensor-triggered runs.
